docs/content/integrations/dbt.mdx

```diff
@@ -1,380 +1,159 @@
-
-```python file=/integrations/dbt.py startafter=start_marker_dbt_asset_mats endbefore=end_marker_dbt_asset_mats dedent=4
-from dagster_dbt.utils import generate_materializations
-
-from dagster import op
-
-@op(required_resource_keys={"dbt"})
-def dbt_run_with_custom_assets(context):
-    dbt_result = context.resources.dbt.run()
-    for materialization in generate_materializations(dbt_result):
-        context.log_event(
-            materialization._replace(
-                metadata_entries=[...]  # insert whatever metadata you want here
-            )
+@repository
+def my_repo():
+    return [
+        dbt_assets,
+        ScheduleDefinition(
+            job=run_something,
+            cron_schedule="@daily",
+        ),
+        ScheduleDefinition(
+            job=run_everything,
+            cron_schedule="@weekly",
         )
-    return dbt_result
+    ]
 ```
 
-**dbt CLI: Set the dbt profile and target to load**
+See (docs for running stuff on a schedule) to learn more.
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_config_profile_and_target endbefore=end_marker_dbt_cli_config_profile_and_target dedent=4
-from dagster_dbt import dbt_cli_resource
-
-from dagster import job
+## Defining Downstream Dependencies
 
-config = {"profile": PROFILE_NAME, "target": TARGET_NAME}
+Dagster allows you to define assets that are downstream of specific dbt models. One property of dbt based assets is that the external tool (in this case, dbt), handles the loading/storing of those models internally, rather than Dagster directly storing/loading the tables that are updated.
 
-@job(resource_defs={"dbt": dbt_cli_resource.configured(config)})
-def my_job():
-    # ...
-```
+This means that there's a range of ways that we might want to load a dbt model as input to a python function. For example, you might want to load the contents of the table as a Pandas dataframe, or you might want to load the contents into a PySpark session.
 
-**dbt CLI: Set the path to the dbt executable**
+To define exactly how you want models to be loaded, you can specify an input manager on the downstream asset.
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_config_executable endbefore=end_marker_dbt_cli_config_executable dedent=4
-from dagster_dbt import dbt_cli_resource
+```python
+from dagster import asset
 
-from dagster import job
-
-config = {"dbt_executable": "path/to/dbt/executable"}
-
-@job(resource_defs={"dbt": dbt_cli_resource.configured(config)})
-def my_job():
-    # ...
+@asset(
+    ins={"model_name": AssetIn(input_manager_key="pandas_dbt_input_manager")}
+)
+def my_downstream_asset(model_name):
+    return
 ```
 
-**dbt CLI: Select specific models to run**
+## Customizing AssetKeys
```

Comment:
This comment is applicable to the mention of AssetKeys in this section as a whole.

What is/are AssetKey(s) referring to, here? Is it the concept of an asset key *or* a literal object or value you'd see in code? Ex:

- Speaking generally/conceptually: "Each asset has an asset key..."
- Referring to a literal object or value in code: "The `AssetKey`..."

Reply:
It's referring to both but I think speaking conceptually is better in this case